core/src/main/java/org/springframework/security/authentication/AbstractUserDetailsReactiveAuthenticationManager.java

```diff
@@ -164,6 +167,11 @@ public void setPostAuthenticationChecks(UserDetailsChecker postAuthenticationChe
 		this.postAuthenticationChecks = postAuthenticationChecks;
 	}
 
+	@Override
```

Comment:
Will you please add JavaDoc for these setters? In the JavaDoc, please add `@since 5.5`.